CHANGES.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+## 1.0.0
+* Creating separate commands for embedding, clustering, and distance matrix creation (pathogen-embed, pathogen-cluster, pathogen-distance) ([28aa891](https://github.com/blab/pathogen-embed/commit/28aa8912a24138a3dcd54fa43f90a5e10b4aa86e))
```

Comment:
For the change logs in Nextstrain, we tend to reference the PR that introduces the change. This allows us to refer to multiple commits with a single link. For an example, [see Augur's change log](https://raw.githubusercontent.com/nextstrain/augur/master/CHANGES.md). I slightly prefer the direct Markdown link approach you use here over the reference-based approach in the Augur change log, but you can use whichever style you prefer as long as it is consistent throughout the log.